lib/factree/dsl.rb

```diff
@@ -0,0 +1,40 @@
+# Readable shortcuts to common functions.
+module Factree::DSL
+  # Creates a {Conclusion} to return from a decision proc.
+  #
+  # The conclusion has an associated value (can be anything.)
+  #
+  # @param [Object] value Any value, including nil
+  # @return [Conclusion]
+  def conclusion(value=nil)
```

Comment:
A decision function should always return a Conclusion, and yeah, you can give the Conclusion a nil value. So far that's only been useful for specs.